.github/workflows/build.yml

```diff
@@ -83,6 +83,14 @@ jobs:
       - name: Build
         run: yarn build
 
+      - name: Cache jest cache
+        uses: actions/cache@56461b9eb0f8438fd15c7a9968e3c9ebb18ceff1 # tag=v3.0.10
+        with:
+          path: .cache/jest
+          key: ${{ runner.os }}-${{ matrix.node-version }}-jest-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ matrix.node-version }}-jest-
```

Comment:
i think this will blow up the cache from time to time because jest isn't removing outdated cache files

Reply:
True, but I think having the hash of `yarn.lock` in there will likely be enough to prevent this.

Reply:
no, the restore key will restore the old cache and so we get old files into new cache

Reply:
Okay, I'll remove the alternative restore-key.